packages/assert/README.md

```diff
@@ -3,8 +3,38 @@ assert
 
 Standard type and schema assertions to be used across all 0x projects and packages
 
-## Install
+## Installation
 
 ```bash
-npm install @0xproject/assert --save
+yarn add @0xproject/assert
 ```
+
+## Usage
+
+TODO - Custom documentation page is in progress.
+
+## Contributing
+
+We strongly encourage our community members to help us make improvements and to determine the future direction of the protocol. To report bugs within this package, please create an issue in this repository.
+
+[CONTRIBUTING.md](../../CONTRIBUTING.md)
+
+## Install Dependencies
```

Comment:
"Install Dependencies"  "Build" "Lint" and "Run Tests" should all be one heading level smaller than "Contributing", according to the quip doc (3 #'s each)